src/auth/auth-token-integration.spec.ts

```diff
@@ -22,6 +22,10 @@ describe('Auth token integration', () => {
   describe('when the location host name is blackbaud.com', () => {
     beforeEach(() => {
       requestSpy = spyOn(BBCsrfXhr, 'request');
+      spyOn(BBAuthGetDomain, 'isRegisteredDomain')
+      .and.returnValue(false);
```

Comment:
The pattern in this code is to add an additional tab after methods that are from objects on the previous line, i.e.:

```
spyOn(BBAuthGetDomain, 'isRegisteredDomain')
  .and.returnValue(false);
```

There are other places in the PR that also need to be updated.